tests/Validator/CMakeLists.txt

```diff
@@ -0,0 +1,61 @@
+if(CMAKE_CONFIGURATION_TYPES)
```

Comment:
I'm afraid this `CMakeLists.txt` needs some changes. I could see that it's probably developed from [tests/POW/CMakeLists.txt](https://github.com/Zilliqa/Zilliqa/blob/de2e47879d620aade220176d0ce9349cc8b7bdf5/tests/POW/CMakeLists.txt). Although it works, it's actually a bit messy and a better CMakeLists.txt example is [tests/Data/CMakeLists.txt](https://github.com/Zilliqa/Zilliqa/blob/master/tests/Data/CMakeLists.txt).